os/arch/arm/src/armv8-m/up_assert.c

```diff
@@ -479,6 +483,22 @@ void up_assert(const uint8_t *filename, int lineno)
 #else
 	lldbg("Assertion failed at file:%s line: %d\n", filename, lineno);
 #endif
+
+#if defined(CONFIG_SCHED_WORKQUEUE) && defined(CONFIG_DEBUG_WORKQUEUE)
+	struct wqueue_s *workq;
+#ifdef CONFIG_SCHED_HPWORK
+	if (strncmp(this_task()->name, HPWORKNAME) == 0) {
+		workq = (struct wqueue_s *)get_hpwork();	
+	}
+#endif
+#ifdef CONFIG_SCHED_LPWORK
+	if (strncmp(this_task()->name, LPWORKNAME) == 0) {
+		workq = (struct wqueue_s *)get_hpwork();
+	}
+#endif
+	lldbg("Running work function is %x, from pid %d.\n", workq->q.head, workq->worker[0].pid);
```

Comment:
1. need checking user workqueue.
2. Even if nothing is matched, this line is executed. This should be printed only when wq is asserted.